src/.env

```diff
@@ -7,7 +7,7 @@ PROJECT_NAME="FastAPI Project"
 STACK_NAME=fastapi-project
 
 # Backend
-BACKEND_CORS_ORIGINS="[\"http://localhost\", \"http://localhost:4200\", \"http://localhost:3000\", \"http://localhost:8080\", \"https://localhost\", \"https://localhost:4200\", \"https://localhost:3000\", \"https://localhost:8080\", \"http://local.dockertoolbox.tiangolo.com\", \"http://localhost.tiangolo.com\"]"
+BACKEND_CORS_ORIGINS="["http://localhost", "http://localhost:4200", "http://localhost:3000", "http://localhost:8080", "https://localhost", "https://localhost:4200", "https://localhost:3000", "https://localhost:8080", "http://local.dockertoolbox.tiangolo.com", "http://localhost.tiangolo.com"]"
```

Comment:
The problem with this is that it should be readable as env vars, if the surrounding quote is double and the internals are also double quotes without escaping, they cancel out, and the result is not valid JSON. Maybe the surrounding quotes could be single quotes.

Doing:

```console
$ source .env
echo $BACKEND_CORS_ORIGINS 
[http://localhost, http://localhost:4200, http://localhost:3000, http://localhost:8080, https://localhost, https://localhost:4200, https://localhost:3000, https://localhost:8080, http://local.dockertoolbox.tiangolo.com, http://localhost.tiangolo.com]
```

...the content of the env var `BACKEND_CORS_ORIGINS` doesn't contain a valid JSON string with an array (list), the values don't have the quotes.

Reply:
fixed